data/gdm-settings.metainfo.xml

```diff
@@ -84,15 +84,15 @@
   <releases>
     <release version="4.3" type="stable" date="2024-02-25">
       <url>https://github.com/gdm-settings/gdm-settings/releases/tag/v4.3</url>
-      <description translatable="no" translate="no">
+      <description translate="no">
```

Comment:
This won't work. See https://github.com/gdm-settings/gdm-settings/pull/238#issuecomment-2031015790 for more info.